documentation/1.0.0/dockerized/scripts/mac-linux/import_forms_mongo.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Extract the MongoDB connection string
```

Comment:
@5Amogh Please update the Readme by including the steps to execute these scripts